app/src/main/java/com/infomaniak/mail/ui/main/folder/ThreadListFragment.kt

```diff
@@ -672,17 +672,19 @@ class ThreadListFragment : TwoPaneFragment(), SwipeRefreshLayout.OnRefreshListen
         val thereAreThreads = (currentThreadsCount ?: 0) > 0
         val filterIsEnabled = mainViewModel.currentFilter.value != ThreadFilter.ALL
         val cursorIsNull = currentFolderCursor == null
-        val isNetworkConnected = mainViewModel.isInternetAvailable.value == true
-        val isBooting = currentThreadsCount == null && !cursorIsNull && isNetworkConnected
-        val shouldDisplayThreadsView = isBooting || thereAreThreads || filterIsEnabled || (cursorIsNull && isNetworkConnected)
+        val networkIsConnected = mainViewModel.isInternetAvailable.value == true
```

Comment:
For almost all the boolean in the app we use the syntax with the verb in first to clearly see it's a boolean, 
it's the 2 others variables here that should be renamed to follow this